### ISSUE TITLE
Component module not found when using with ember-cli v3.13.0 with ember-bootstrap

### ISSUE THREAD

**Issue by veebs** [kaliber5/ember-bootstrap#881]
---

Output from `ember bootstrap:info`:

```
PS C:\dev\svu-ui-poc\ember-app> ember bootstrap:info
Npm packages:
ember-bootstrap: ^3.0.0-rc.2 -> 3.0.0-rc.2
ember-cli: github:ember-cli/ember-cli#a07650e8465fc8abe35a4a6f269b0f88d047ec58 -> 3.13.0-beta.1
bootstrap: ^4.3.1 -> 4.3.1
bootstrap-sass: n/a
ember-cli-sass: n/a
ember-cli-less: n/a
Bower packages:
bootstrap: n/a
bootstrap-sass: n/a
ember-bootstrap configuration:
bootstrapVersion: 4
importBootstrapCSS: true
```

Components are being registered with ember as `ember-bootstrap/templates/common/components/bs-alert` rather than  `ember-bootstrap/templates/components/bs-alert`.

This is because the `treeForAddonTemplates` hook which is no longer invoked as of [#8633.](https://github.com/ember-cli/ember-cli/pull/8633)

See discussions [here](https://github.com/ember-cli/ember-cli/issues/8693)

The end result is the template folder does not get a clean up and contains the bs3, bs4 and common folders.

![image](https://user-images.githubusercontent.com/367073/63337365-37c0ea80-c384-11e9-9a6c-2210d27dd20c.png)

A quick fix is to move the code form `treeForAddonTemplates()`  to `treeForAddonTemplates()` in index.js.

```javascript
  treeForAddon(tree) {
    let bsVersion = this.getBootstrapVersion();
    let otherBsVersion = this.getOtherBootstrapVersion();
    let componentsPath = 'components/';

    tree = this.debugTree(tree, 'addon-tree:input');
    tree = mv(tree, `${componentsPath}bs${bsVersion}/`, componentsPath);
    tree = rm(tree, `${componentsPath}bs${otherBsVersion}/**/*`);

    tree = this.debugTree(tree, 'addon-tree:bootstrap-version');
    tree = this.filterComponents(tree);
    tree = this.debugTree(tree, 'addon-tree:tree-shaken');

    // START - Moved from treeForAddonTemplates()
    let templatePath = 'templates/components/';

    tree = this.debugTree(tree, 'addon-templates-tree:input');
    tree = mv(tree, `${templatePath}common/`, templatePath);
    tree = mv(tree, `${templatePath}bs${bsVersion}/`, templatePath);
    tree = rm(tree, `${templatePath}bs${otherBsVersion}/**/*`);

    tree = this.debugTree(tree, 'addon-templates-tree:bootstrap-version');
    tree = this.filterComponents(tree);
    tree = this.debugTree(tree, 'addon-templates-tree:tree-shaken');
    // END - Moved from treeForAddonTemplates()

    return this._super.treeForAddon.call(this, tree);
  },

```

**Comment by simonihmig** [comment]
Thanks for the report @veebs! Seems they acknowledged it's a bug on their side, but we should nevertheless make the changes here, if that hook is going to end as deprecated. As you already have laid out the solution, would you be interested in providing a PR?

**Comment by veebs** [comment]
No problems. PR coming soon.

**Comment by veebs** [comment]
PR - https://github.com/kaliber5/ember-bootstrap/pull/883

**Comment by jelhan** [comment]
Ember CLI v3.13.0 got released without a fix. :sob:

**Comment by simonihmig** [comment]
Closed by #883

**Comment by jelhan** [comment]
Reopen as we should consider backporting this one to v2.

**Comment by simonihmig** [comment]
I was thinking about the same seconds ago 👍

**Comment by simonihmig** [comment]
Fixed in v2.8.1! :tada:
